### ISSUE TITLE
usage template-helpers in assemble

### ISSUE THREAD

**Issue by onokumus** [jonschlinkert/template-helpers#2]
Hi.

I failed to use this package in my assemble project. 

``` json
"devDependencies": {
  "assemble": "^0.9.0",
  "gulp-extname": "^0.2.2",
  "template-helpers": "^0.5.1"
}
```

my `assemblefile.js`

``` js
var assemble = require('assemble');
var extname = require('gulp-extname');

var app = assemble();

app.task('load', function(cb) {
  app.layouts('templates/layouts/**/*.hbs');
  app.partials('templates/partials/**/*.hbs');
  app.pages('content/*.hbs');

  var helpers = require('template-helpers')('math');
  var math = {math: helpers};
  app.helper('add', math);
  cb();
});

app.task('content', ['load'], function() {
  return app.toStream('pages')
    .pipe(app.renderFile())
    .on('err', console.error)
    .pipe(extname())
    .pipe(app.dest('dist'));
});

app.task('watch', function() {
  app.watch('**/*.hbs', ['content']);
});

app.task('default', ['content'], app.parallel(['watch']));

module.exports = app;
```

content

``` hbs
{{add 5 4}}
```

but 

``` bash
[Error: Missing helper: "add"]
....
```

**Comment by jonschlinkert** [comment]
In this code:

``` js
var helpers = require('template-helpers')('math');
var math = {math: helpers};
app.helper('add', math);
```

The code in your example will try to load all of the `math` helpers as one helper: `add`. 

Try doing either:

``` js
var math = require('template-helpers')('math');
app.helper('add', math.add);
```

Or 

``` js
var math = require('template-helpers')('math');
app.helpers(math);
```

**Comment by onokumus** [comment]
if i use it, OK

``` js
var math = require('template-helpers')('math');
app.helper('add', math.add);
```

but, if this

``` js
var math = require('template-helpers')('math');
app.helpers(math);
```

konsole print

``` bash

{ [RangeError: Maximum call stack size exceeded] undefined: true }
events.js:154
      throw er; // Unhandled 'error' event
      ^

RangeError: Maximum call stack size exceeded
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:14:18)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)
    at deepBind (/home/onokumus/bat27ocak/assemble-collection/node_modules/deep-bind/index.js:24:21)

```

**Comment by jonschlinkert** [comment]
thanks, I'll look into it

**Comment by jonschlinkert** [comment]
closing since this I believe this was resolved by 0.5.1 but I must have forgotten to resolve this issue in the commit. please reopen if you're still having problems
